bee-network/bee-autopeering/src/init.rs

```diff
@@ -162,12 +167,14 @@ where
 
     // Reverify old peers regularly.
     let f = query::reverify_fn();
-    let delay = iter::repeat(DEFAULT_REVERIFY_INTERVAL);
+    let delay = iter::repeat(BOOTSTRAP_VERIFICATION_DELAY)
+        .take(BOOTSTRAP_MAX_VERIFICATIONS.min(active_peers.read().len()))
+        .chain(iter::repeat(REVERIFY_INTERVAL_DEFAULT));
```

Comment:
This took some time to wrap my head around 🙃 , but I wouldn't know a cleaner way to do it 👍.

Reply:
yea, I can understand the confusion. Maybe I should add a comment explaining why that works. (the peers live in a ring, and sending a verification request moves the ring by 1, so `BOOTSTRAP_VERIFICATION_DELAY` will be used for the maximum of 10 peers. After that the normal reverification interval applies.